frontend/src/pages/pipelines/global/experiments/compareRuns/CompareRunParamsSection.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateHeader,
+  EmptyStateVariant,
+  ExpandableSection,
+  Flex,
+  Switch,
+} from '@patternfly/react-core';
+import { InnerScrollContainer, TableVariant, Td, Tr } from '@patternfly/react-table';
+
+import { SortableData, Table } from '~/components/table';
+import { useCompareRuns } from '~/concepts/pipelines/content/compareRuns/CompareRunsContext';
+import { RuntimeConfigParamValue } from '~/concepts/pipelines/kfTypes';
+import { normalizeInputParamValue } from '~/concepts/pipelines/content/pipelinesDetails/pipelineRun/utils';
+
+export const CompareRunParamsSection: React.FunctionComponent = () => {
+  const { loaded, selectedRuns } = useCompareRuns();
+  const [isSectionOpen, setIsSectionOpen] = React.useState(true);
+  const [isHideSameRowsChecked, setIsHideSameRowsChecked] = React.useState<boolean>(false);
+  const hasSelectedRuns = !!selectedRuns.length;
+
+  const runNameColumns: SortableData<[string, Record<string, RuntimeConfigParamValue>]>[] =
+    React.useMemo(
+      () =>
+        hasSelectedRuns
+          ? [
+              {
+                label: 'Run name',
+                field: 'run-name',
+                isStickyColumn: true,
+                hasRightBorder: true,
+                backgroundColor: '#ededed',
```

Comment:
I am pretty sure there is not a way to do this through patternfly, so first i want to double check with @yannnz @xianli123 -- PF does not allow us to set individual columns a certain darker color (only alternating rows). Is this something that is needed? If so we can continue using the backgroundColor custom styling. If not we should remove it in favor of staying more consistent with PF.

Reply:
I meant to switch this to a PF className, so I've updated with that instead. This was demoed to both Yan & Kun a couple days ago and the idea is that we contribute this guideline back to them at some point for use-cases similar to ours.

Reply:
Ok. Do you mind opening up a upstream patternfly feature request ticket with the use case. Then comment that ticket url in the scss file next to where the class is defined for the custom styling as a TODO. This is how we have been tracking upstream PF tickets so we can go back and remove them when they close

Reply:
It's not a custom class name, it's a PF class name that we're using here. Yan started a conversation here: https://patternfly.slack.com/archives/CGR204RM2/p1712631660224649.

I can open a discussion in github and link that in a comment where the patternfly className is used, but beyond that, this has yet to be deemed something PF would work on.

Reply:
ok. yeah then that is fine. If an issue does get made we should make sure to document it somewhere. in the meantime, this doesnt need to block the pr